imports/plugins/included/payments-stripe/register.js

```diff
@@ -2,12 +2,20 @@
 import Reaction from "/imports/plugins/core/core/server/Reaction";
 import resolvers from "./server/no-meteor/resolvers";
 import schemas from "./server/no-meteor/schemas";
+import stripeCapturePayment from "./server/no-meteor/util/stripeCapturePayment";
+import stripeCreateRefund from "./server/no-meteor/util/stripeCreateRefund";
+import stripeListRefund from "./server/no-meteor/util/stripeListRefund";
 
 Reaction.registerPackage({
   label: "Stripe",
   name: "reaction-stripe",
   icon: "fa fa-cc-stripe",
   autoEnable: true,
+  functionsByType: {
+    stripeCapturePayment: [stripeCapturePayment],
+    stripeCreateRefund: [stripeCreateRefund],
+    stripeListRefund: [stripeListRefund]
+  },
```

Comment:
Instead of this, add a `functions` object to the `paymentMethods` object below.

```js
paymentMethods: [{
  name: "stripe_card",
  displayName: "Stripe Card",
  functions: {
    capturePayment: stripeCapturePayment,
    createRefund: stripeCreateRefund,
    listRefund: stripeListRefund
  }
}],
```

Then when you need to use one of the functions, you can get the registered payment method by name based on what's in the order payment data, and call `paymentMethod.functions.capturePayment()` for example.